src/server.js

```diff
@@ -211,6 +211,46 @@ app.get(
   }
 );
 
+function averageGeolocation(coords) {
+  if (coords.length === 1) {
+    return {
+      latitude: coords[0].lat,
+      longitude: coords[0].long,
+    };
+  }
+  let x = 0.0;
+  let y = 0.0;
+  let z = 0.0;
+
+  if (coords.length < 1) {
```

Comment:
this is mixing styles and logic in a weird way:

1) humans naturally count upwards, so why handle cases in order [1, 0, N] when reading through this code? it is easier to understand and follow if you handle them in order [0, 1, N].
2) the first case [1] returns early and does not have an else statement. the second case [0] returns early and does have an else statement? changing styles within the same function makes code harder to read, so it's preferred to pick a style and stick to it.

Reply:
I wasn't actually thinking of things this way. I learnt that it's good practice to check for edge cases first before diving into the best case. What your'e saying makes a lot of sense. I'll work on it.

Reply:
checking edge cases first is good practice; here you have two edge cases [0, 1] and the rest of the cases. with those two edge cases, you might as well put them in natural order and organize the code the same for both.